1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,208 @@
+---
+post_title: Quick Start
+feature_maturity: experimental
+menu_order: 3.3
+---
+
+Use this guide to get started with the `dcos task exec` debugging command.
+
+**Prerequisite:**
+
+- A container launched by using the [DC/OS Universal container runtime](/docs/1.9/usage/containerizers/).
+
+# Pipe output from a command running inside a container 
+
+You can run commands inside a container by using the `dcos task exec` command. In this example, a long running Marathon app is launched and then the `dcos task exec` command is used to get the hostname of the node running the app. 
+
+1.  Create a Marathon app definition and name it `my-app.json` with the following contents:
+
+    ```bash
+    {
+       "id": "/my-app",
+       "cmd": "sleep 100000000",
+       "cpus": 1,
+       "instances": 1
+     }
+     ```
+
+1.  Deploy the service on DC/OS:
+
+    ```bash
+    $ dcos marathon app add my-app.json
+    ```
+
+1.  Get the task ID of the job with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME        HOST        USER  STATE  ID                                               
+    my-app  10.0.1.106  root    R    <task_id>
+    ```
+
+1.  Run this command to show the hostname of the container running your app, where `<task-ID>` is your task ID.
+
+    ```bash
+    $ dcos task exec <task_id> hostname
+    ```
+    
+    The output should look similar to this:
+    
+    ```bash
+    ip-10-0-1-105.us-west-2.compute.internal
+    ```
+
+For more information about the `dcos task exec` command, see the CLI command [reference](/docs/1.9/usage/cli/command-reference/).
+
+# Run an interactive command inside task container
+You can run interactive commands on machines in your cluster by using the `dcos task exec` command. In this example, the `dcos task exec` command is used to copy a simple script from your local machine to the task container on the node. The script is then administered locally by using the `dcos task exec` command.
+
+1.  Create a Marathon app definition and name it `my-interactive-app.json` with the following contents:
+
+    ```bash
+    {
+       "id": "/my-interactive-app",
+       "cmd": "sleep 100000000",
+       "cpus": 1
+    }
```

Comment:
The app definition for "/my-app" has `"instances": 1` set in it. I'd either add this here or remove it from above (the default is 1 if you don't specify anything). My preference is to remove it from above to make these definitions as simple as possible.